graylog2-web-interface/src/views/components/DashboardSearchBar.test.tsx

```diff
@@ -38,8 +38,13 @@ jest.mock('views/stores/GlobalOverrideStore', () => ({
 }));
 
 jest.mock('views/stores/SearchStore', () => ({
+  SearchStore: MockStore(
+    'listen',
+    ['getInitialState', () => ({ search: { parameters: [] } })],
+  ),
   SearchActions: {
     refresh: jest.fn(),
+    triggerExecutionAttempt: { completed: { listen: () => () => {} } },
```

Comment:
We can also use this:
```suggestion
    triggerExecutionAttempt: mockAction(),
```